src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -449,26 +385,25 @@ public void canUpload(String name, long fileSize) {
         URL url = GET_ITEMS_URL.build(this.getAPI().getBaseURL(), BoxFolder.this.getID());
         return new BoxItemIterator(BoxFolder.this.getAPI(), url);
     }
-
     /**
-     * Searches this folder and all descendant folders using a given query.
+     * Function is Deprecated, please use "BoxSearch Class instead".
```

Comment:
I'd suggest rephrasing this a bit and linking to the BoxSearch class:

    "This method is deprecated, please use the {@link BoxSearch} class instead.